src/content/peerconnection/endtoend-encryption/js/main.js

```diff
@@ -110,6 +112,19 @@ function dump(chunk, direction, max = 16) {
   console.log(performance.now().toFixed(2), direction, bytes.trim(), chunk.data.byteLength, (data[0] & 0x1) === 0);
 }
 
+// If using crypto offset:
+// Do not encrypt the first bytes of the payload. For VP8
+// this is the content described in
+//   https://tools.ietf.org/html/rfc6386#section-9.1
+// which is 10 bytes for key frames and 3 bytes for delta frames.
+// For opus (where chunk.type is not set) this is the TOC byte from
+//   https://tools.ietf.org/html/rfc6716#section-3.1
```

Comment:
Add a description of the purpose, something like:

This allows the middlebox to decode some packets without crashing, making for more interesting pictures and sounds.